### ISSUE TITLE
sitewide 508-defect-4 [COGNITION, SCREENREADER]: CONSIDER announcing empty fields to SR users in the review accordion component

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#16892]
# [508-defect-4](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-4)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

<hr/>

## Point of Contact

**VFS Point of Contact:** Josh Kim

## User Story or Problem Statement

As a screen reader user, I expect to know when a field is empty so that I can make edits confidently while reviewing my information.

## Details

Currently, the expanded review accordion is a definition list. VoiceOver will announce the `dt` along with its placement in the order, skip over the empty `dd` element, and continue to the following `dt` — e.g. Middle Name, 3 of 12 --> Last Name, 5 of 12. NVDA will not announce their order, and will simply skip empty `dd` elements.

Screen reader users may miss that a field is empty if it is not clearly announced.

## Acceptance Criteria

- [ ] Empty fields are announced to screen reader users.

## Environment

* Operating System: Mac, Windows
* Browser: Safari, Firefox
* Screenreading device: VoiceOver, NVDA
* Server destination: staging, production

## Steps to Recreate

1. Fill out Form 28-8832 as an unsigned in user and reach the review application page.
2. Start screenreading device listed in Environment.
3. Expand the applicant information accordion.
4. Continue through the applicant information and confirm VoiceOver or NVDA skips all empty `dd` fields.

## Solution (if known)

Two solutions may exist:
- Add a screen reader only span stating, "did not answer" or "no response".
- Add a visual, filler span for empty items e.g. "did not answer" or "no response".

I recommend working with content to determine an appropriate message.

### SR-only
```html
<dd>
 <span class="sr-only">No response</span>
</dd>
```

### Visual
```html
<dd>
 <span>No response</span>
</dd>
```

## Screenshots or Trace Logs
Current version has a completely empty `dd` element.
![image](https://user-images.githubusercontent.com/14154792/101068196-67d6d180-3566-11eb-856f-e19c851ea6a1.png)

**Comment by jason-gcio** [comment]
This must be done at the Form System level...we can do this, I think, but it is outside our scope at this time.

**Comment by jason-gcio** [comment]
Completed in another ticket
